<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Publications | Xiang Li</title><meta name=keywords content><meta name=description content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.

Selected Publications


Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040


Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790


A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866


Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944"><meta name=author content><link rel=canonical href=https://glefhoho.github.io/publications/><link crossorigin=anonymous href=/assets/css/stylesheet.36819bea596090d8b48cf10d9831382996197aa7e4fc86f792f7c08c9ca4d23b.css integrity="sha256-NoGb6llgkNi0jPENmDE4KZYZeqfk/Ib3kvfAjJyk0js=" rel="preload stylesheet" as=style><link rel=icon href=https://glefhoho.github.io/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://glefhoho.github.io/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://glefhoho.github.io/favicon-32x32.png><link rel=apple-touch-icon href=https://glefhoho.github.io/apple-touch-icon.png><link rel=mask-icon href=https://glefhoho.github.io/safari-pinned-tab.svg><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><link rel=alternate hreflang=en href=https://glefhoho.github.io/publications/><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--code-block-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:url" content="https://glefhoho.github.io/publications/"><meta property="og:site_name" content="Xiang Li"><meta property="og:title" content="Publications"><meta property="og:description" content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.
Selected Publications Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040
Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790
A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866
Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944"><meta property="og:locale" content="en-us"><meta property="og:type" content="article"><meta property="article:published_time" content="2025-08-04T00:00:00+00:00"><meta property="article:modified_time" content="2025-08-04T00:00:00+00:00"><meta name=twitter:card content="summary"><meta name=twitter:title content="Publications"><meta name=twitter:description content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.

Selected Publications


Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040


Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790


A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866


Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Publications","item":"https://glefhoho.github.io/publications/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Publications","name":"Publications","description":"For a full list of my peer-reviewed publications, please visit my Google Scholar profile.\nSelected Publications Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains\nFEMS Microbiol. Lett. (2025)\nhttps://doi.org/10.1093/femsle/fnaf040\nMetagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment\nACS ES\u0026amp;T Water 4 (12), 5800-5810 (2024)\nhttps://doi.org/10.1021/acsestwater.4c00790\nA novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data\nMicroorganisms 12 (9), 1866 (2024)\nhttps://doi.org/10.3390/microorganisms12091866\nVirus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study\nJ. Hazard. Mater. 458, 131944 (2023)\nhttps://doi.org/10.1016/j.jhazmat.2023.131944\n","keywords":[],"articleBody":"For a full list of my peer-reviewed publications, please visit my Google Scholar profile.\nSelected Publications Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains\nFEMS Microbiol. Lett. (2025)\nhttps://doi.org/10.1093/femsle/fnaf040\nMetagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment\nACS ES\u0026T Water 4 (12), 5800-5810 (2024)\nhttps://doi.org/10.1021/acsestwater.4c00790\nA novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data\nMicroorganisms 12 (9), 1866 (2024)\nhttps://doi.org/10.3390/microorganisms12091866\nVirus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study\nJ. Hazard. Mater. 458, 131944 (2023)\nhttps://doi.org/10.1016/j.jhazmat.2023.131944\nIn situ Nanopore sequencing reveals metabolic characteristics of the Qilian glacier meltwater microbiome\nEnviron Sci Pollut Res 30 (35), 84805–84813 (2023)\nhttps://doi.org/10.1007/s11356-023-28250-0\nPlastic materials and water sources actively select and shape wastewater plastisphere over time\nFront Environ Sci Eng 16(11): 145 (2022)\nhttps://doi.org/10.1007/s11783-022-1580-1\nDeveloping a Bifidobacterium phage qPCR-based assay for tracking sewage pollution\nSci. Total Environ. 838, 155815 (2022)\nhttps://doi.org/10.1016/j.scitotenv.2022.155815\nMetagenomic and viromic data mining reveals viral threats in biologically treated domestic wastewater\nEnviron. Sci. Ecotechnol. 100105 (2021)\nhttps://doi.org/10.1016/j.ese.2021.100105\nA metagenomic-based method to study hospital air dust resistome\nChem. Eng. J. 406, 126854 (2021)\nhttps://doi.org/10.1016/j.cej.2020.126854\nVariable fecal source prioritization in recreational waters monitored with viral and bacterial indicators\nWater Res. 116845 (2021)\nhttps://doi.org/10.1016/j.watres.2021.116845\nLarge-scale implementation of standardized qPCR fecal source ID in Tillamook Bay Watershed\nPLoS ONE 14(6): e0216827 (2019)\nhttps://doi.org/10.1371/journal.pone.0216827\nA novel MST microarray for pathogen detection and source identification in environmental systems\nEnviron. Sci. Technol. 49(12): 7319–7329 (2015)\nhttps://doi.org/10.1021/acs.est.5b00980\n","wordCount":"244","inLanguage":"en","datePublished":"2025-08-04T00:00:00Z","dateModified":"2025-08-04T00:00:00Z","mainEntityOfPage":{"@type":"WebPage","@id":"https://glefhoho.github.io/publications/"},"publisher":{"@type":"Organization","name":"Xiang Li","logo":{"@type":"ImageObject","url":"https://glefhoho.github.io/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://glefhoho.github.io/ accesskey=h title="Xiang Li (Alt + H)">Xiang Li</a><div class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)" aria-label="Toggle theme">
<svg id="moon" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg>
<svg id="sun" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></div></div><ul id=menu><li><a href=https://glefhoho.github.io/about/ title=About><span>About</span></a></li><li><a href=https://glefhoho.github.io/publications/ title=Publications><span class=active>Publications</span></a></li><li><a href=https://glefhoho.github.io/talks/ title=Talks><span>Talks</span></a></li><li><a href=https://glefhoho.github.io/projects/ title=Projects><span>Projects</span></a></li><li><a href=https://glefhoho.github.io/teaching/ title=Teaching><span>Teaching</span></a></li><li><a href=https://glefhoho.github.io/posts/ title=Posts><span>Posts</span></a></li></ul></nav></header><main class=main><article class=post-single><header class=post-header><h1 class="post-title entry-hint-parent">Publications</h1><div class=post-meta><span title='2025-08-04 00:00:00 +0000 UTC'>August 4, 2025</span></div></header><div class=post-content><p>For a full list of my peer-reviewed publications, please visit my <a href="https://scholar.google.com/citations?user=wPsBNOsAAAAJ&amp;hl=en">Google Scholar profile</a>.</p><hr><h3 id=selected-publications>Selected Publications<a hidden class=anchor aria-hidden=true href=#selected-publications>#</a></h3><ul><li><p><strong>Using core genome and machine learning for serovar prediction in <em>Salmonella enterica</em> subspecies I strains</strong><br><em>FEMS Microbiol. Lett.</em> (2025)<br><a href=https://doi.org/10.1093/femsle/fnaf040>https://doi.org/10.1093/femsle/fnaf040</a></p></li><li><p><strong>Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment</strong><br><em>ACS ES&amp;T Water</em> 4 (12), 5800-5810 (2024)<br><a href=https://doi.org/10.1021/acsestwater.4c00790>https://doi.org/10.1021/acsestwater.4c00790</a></p></li><li><p><strong>A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data</strong><br><em>Microorganisms</em> 12 (9), 1866 (2024)<br><a href=https://doi.org/10.3390/microorganisms12091866>https://doi.org/10.3390/microorganisms12091866</a></p></li><li><p><strong>Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study</strong><br><em>J. Hazard. Mater.</em> 458, 131944 (2023)<br><a href=https://doi.org/10.1016/j.jhazmat.2023.131944>https://doi.org/10.1016/j.jhazmat.2023.131944</a></p></li><li><p><strong>In situ Nanopore sequencing reveals metabolic characteristics of the Qilian glacier meltwater microbiome</strong><br><em>Environ Sci Pollut Res</em> 30 (35), 84805–84813 (2023)<br><a href=https://doi.org/10.1007/s11356-023-28250-0>https://doi.org/10.1007/s11356-023-28250-0</a></p></li><li><p><strong>Plastic materials and water sources actively select and shape wastewater plastisphere over time</strong><br><em>Front Environ Sci Eng</em> 16(11): 145 (2022)<br><a href=https://doi.org/10.1007/s11783-022-1580-1>https://doi.org/10.1007/s11783-022-1580-1</a></p></li><li><p><strong>Developing a <em>Bifidobacterium</em> phage qPCR-based assay for tracking sewage pollution</strong><br><em>Sci. Total Environ.</em> 838, 155815 (2022)<br><a href=https://doi.org/10.1016/j.scitotenv.2022.155815>https://doi.org/10.1016/j.scitotenv.2022.155815</a></p></li><li><p><strong>Metagenomic and viromic data mining reveals viral threats in biologically treated domestic wastewater</strong><br><em>Environ. Sci. Ecotechnol.</em> 100105 (2021)<br><a href=https://doi.org/10.1016/j.ese.2021.100105>https://doi.org/10.1016/j.ese.2021.100105</a></p></li><li><p><strong>A metagenomic-based method to study hospital air dust resistome</strong><br><em>Chem. Eng. J.</em> 406, 126854 (2021)<br><a href=https://doi.org/10.1016/j.cej.2020.126854>https://doi.org/10.1016/j.cej.2020.126854</a></p></li><li><p><strong>Variable fecal source prioritization in recreational waters monitored with viral and bacterial indicators</strong><br><em>Water Res.</em> 116845 (2021)<br><a href=https://doi.org/10.1016/j.watres.2021.116845>https://doi.org/10.1016/j.watres.2021.116845</a></p></li><li><p><strong>Large-scale implementation of standardized qPCR fecal source ID in Tillamook Bay Watershed</strong><br><em>PLoS ONE</em> 14(6): e0216827 (2019)<br><a href=https://doi.org/10.1371/journal.pone.0216827>https://doi.org/10.1371/journal.pone.0216827</a></p></li><li><p><strong>A novel MST microarray for pathogen detection and source identification in environmental systems</strong><br><em>Environ. Sci. Technol.</em> 49(12): 7319–7329 (2015)<br><a href=https://doi.org/10.1021/acs.est.5b00980>https://doi.org/10.1021/acs.est.5b00980</a></p></li></ul></div><footer class=post-footer><ul class=post-tags></ul></footer></article></main><footer class=footer><span>&copy; 2025 <a href=https://glefhoho.github.io/>Xiang Li</a></span> ·
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
<a href=https://github.com/adityatelange/hugo-PaperMod/ rel=noopener target=_blank>PaperMod</a></span></footer><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg viewBox="0 0 12 6" fill="currentColor"><path d="M12 6H0l6-6z"/></svg>
</a><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>