<!DOCTYPE html>
<html><head lang="en">
	<meta charset="utf-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge"><title>Publications - </title><meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.

Selected Publications
Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040
Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790
A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866
Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944" />
	<meta property="og:image" content=""/>
	<meta property="og:url" content="https://glefhoho.github.io/publications/">
  <meta property="og:title" content="Publications">
  <meta property="og:description" content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.
Selected Publications
Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040
Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790
A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866
Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:published_time" content="2025-08-29T00:00:00+00:00">
    <meta property="article:modified_time" content="2025-08-29T00:00:00+00:00">

  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="Publications">
  <meta name="twitter:description" content="For a full list of my peer-reviewed publications, please visit my Google Scholar profile.
Selected Publications
Using core genome and machine learning for serovar prediction in Salmonella enterica subspecies I strains
FEMS Microbiol. Lett. (2025)
https://doi.org/10.1093/femsle/fnaf040
Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment
ACS ES&amp;T Water 4 (12), 5800-5810 (2024)
https://doi.org/10.1021/acsestwater.4c00790
A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data
Microorganisms 12 (9), 1866 (2024)
https://doi.org/10.3390/microorganisms12091866
Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study
J. Hazard. Mater. 458, 131944 (2023)
https://doi.org/10.1016/j.jhazmat.2023.131944">

        <link href="https://glefhoho.github.io/css/fonts.11a1877508139eac0b5b4852ceb110c35641b3533321e66e39149e901ed5756b.css" rel="stylesheet">
	

	
	<link rel="stylesheet" type="text/css" media="screen" href="https://glefhoho.github.io/css/main.e08e3d47ef734c0d298ed0fb46ed50640341570139eb362f34ac2720f01a4f58.css" />
</head>
<body>
        <div class="content"><header>
	<div class="main">
		<a href="https://glefhoho.github.io/"></a>
	</div>
	<nav>
		
		<a href="/about/">About Me</a>
		
		<a href="/publications/">Publications</a>
		
		<a href="/talks/">Talks</a>
		
		<a href="/projects/">Projects</a>
		
		<a href="/teaching/">Teaching</a>
		
		
	</nav>
</header>

<main>
  <article>
    <div class="post-container">
      
      <div class="post-content">
        <div class="title">
          <h1 class="title">Publications</h1>
          <div class="meta">Posted on Aug 29, 2025</div>
        </div>
        
        <section class="body">
          <p>For a full list of my peer-reviewed publications, please visit my <a href="https://scholar.google.com/citations?user=wPsBNOsAAAAJ&amp;hl=en">Google Scholar profile</a>.</p>
<hr>
<p><strong>Selected Publications</strong></p>
<p><strong>Using core genome and machine learning for serovar prediction in <em>Salmonella enterica</em> subspecies I strains</strong><br>
<em>FEMS Microbiol. Lett.</em> (2025)<br>
<a href="https://doi.org/10.1093/femsle/fnaf040">https://doi.org/10.1093/femsle/fnaf040</a></p>
<p><strong>Metagenomics evidence of a novel Thermoplasmata-like archaeon decomposing heterodisulfide in nonacidic estuary-ocean sediment</strong><br>
<em>ACS ES&amp;T Water</em> 4 (12), 5800-5810 (2024)<br>
<a href="https://doi.org/10.1021/acsestwater.4c00790">https://doi.org/10.1021/acsestwater.4c00790</a></p>
<p><strong>A novel Slope-Matrix-Graph algorithm to analyze compositional microbiome data</strong><br>
<em>Microorganisms</em> 12 (9), 1866 (2024)<br>
<a href="https://doi.org/10.3390/microorganisms12091866">https://doi.org/10.3390/microorganisms12091866</a></p>
<p><strong>Virus-impacted community adaptation in oligotrophic groundwater revealed by Hi-C coupled metagenomic and viromic study</strong><br>
<em>J. Hazard. Mater.</em> 458, 131944 (2023)<br>
<a href="https://doi.org/10.1016/j.jhazmat.2023.131944">https://doi.org/10.1016/j.jhazmat.2023.131944</a></p>
<p><strong>In situ Nanopore sequencing reveals metabolic characteristics of the Qilian glacier meltwater microbiome</strong><br>
<em>Environ Sci Pollut Res</em> 30 (35), 84805–84813 (2023)<br>
<a href="https://doi.org/10.1007/s11356-023-28250-0">https://doi.org/10.1007/s11356-023-28250-0</a></p>
<p><strong>Plastic materials and water sources actively select and shape wastewater plastisphere over time</strong><br>
<em>Front Environ Sci Eng</em> 16(11): 145 (2022)<br>
<a href="https://doi.org/10.1007/s11783-022-1580-1">https://doi.org/10.1007/s11783-022-1580-1</a></p>
<p><strong>Developing a <em>Bifidobacterium</em> phage qPCR-based assay for tracking sewage pollution</strong><br>
<em>Sci. Total Environ.</em> 838, 155815 (2022)<br>
<a href="https://doi.org/10.1016/j.scitotenv.2022.155815">https://doi.org/10.1016/j.scitotenv.2022.155815</a></p>
<p><strong>Metagenomic and viromic data mining reveals viral threats in biologically treated domestic wastewater</strong><br>
<em>Environ. Sci. Ecotechnol.</em> 100105 (2021)<br>
<a href="https://doi.org/10.1016/j.ese.2021.100105">https://doi.org/10.1016/j.ese.2021.100105</a></p>
<p><strong>A metagenomic-based method to study hospital air dust resistome</strong><br>
<em>Chem. Eng. J.</em> 406, 126854 (2021)<br>
<a href="https://doi.org/10.1016/j.cej.2020.126854">https://doi.org/10.1016/j.cej.2020.126854</a></p>
<p><strong>Variable fecal source prioritization in recreational waters monitored with viral and bacterial indicators</strong><br>
<em>Water Res.</em> 116845 (2021)<br>
<a href="https://doi.org/10.1016/j.watres.2021.116845">https://doi.org/10.1016/j.watres.2021.116845</a></p>
<p><strong>Large-scale implementation of standardized qPCR fecal source ID in Tillamook Bay Watershed</strong><br>
<em>PLoS ONE</em> 14(6): e0216827 (2019)<br>
<a href="https://doi.org/10.1371/journal.pone.0216827">https://doi.org/10.1371/journal.pone.0216827</a></p>
<p><strong>A novel MST microarray for pathogen detection and source identification in environmental systems</strong><br>
<em>Environ. Sci. Technol.</em> 49(12): 7319–7329 (2015)<br>
<a href="https://doi.org/10.1021/acs.est.5b00980">https://doi.org/10.1021/acs.est.5b00980</a></p>

        </section>
        <div class="post-tags">
          
        </div>
      </div>

      
      
    </div>

    </article>
</main>
<footer>
  <div style="display:flex"></div>
  <div class="footer-info">
    2025  <a
      href="https://github.com/athul/archie">Archie Theme</a> | Built with <a href="https://gohugo.io">Hugo</a>
  </div>
</footer>



</div>
    </body>
</html>
